app/database/database_setup.py

```diff
@@ -0,0 +1,124 @@
+import sqlite3
+
+from app.util.conf.configuration import Configuration
+
+
+class FailedSQLiteTableSetup(Exception):
+    pass
+
+
+class DatabaseSetup:
```

Comment:
Yeah 😅 I made this class before we started using SQLAlchemy and didn't realize you could create tables with it, we can definitely do away with this entire class